src/gpu/intel/jit/gemm/generator/pieces/copy.cxx

```diff
@@ -246,7 +246,9 @@ void Generator<hw>::copyExecute(CopyPlan &&plan, CommonState &state)
         if (clobbered[2*i] || clobbered[2*i + 1])
             mov(1, savedFlags[i], FlagRegister(i));
 
-    // Generate code.
+    sync(SyncFunction::nop, SWSB<uint32_t>(1));
```

Comment:
Is it a workaround? Do you think it's a hardware bug?
Would be nice to have a comment in the code.

Reply:
Agreed, I really would rather not add a sync without knowing the underlying problem. Also, there are possible performance penalties.

Reply:
Perf looks OK to me, I'm OK with merging this as long as we create a Jira or gemmstone GitHub issue to track a proper solution.